ReportUtilityBills/measure.rb

```diff
@@ -677,8 +677,9 @@ def setup_utility_outputs()
   def get_outputs(fuels, utility_bill_scenario)
     fuels.each do |(fuel_type, _is_production), fuel|
       unit_conv = UnitConversions.convert(1.0, 'J', fuel.units)
-      unit_conv /= 139.0 if fuel_type == FT::Oil
-      unit_conv /= 91.6 if fuel_type == FT::Propane
+      unit_conv /= 139.0 if fuel_type == FT::Oil # Convert kBtu to gallons of oil
+      unit_conv /= 91.6 if fuel_type == FT::Propane # Convert kBtu to gallons of propane
+      unit_conv /= 1000.0 if [FT::Coal, FT::WoodCord, FT::WoodPellets].include? fuel_type # Convert kBtu to MMBtu
```

Comment:
Use `UnitConversions` here too. Having the same numbers two different places in the code can easily lead to bugs where someone in the future updates one but not the other. And MBtu instead of MMBtu again.